api/src/main/resources/messages.properties

```diff
@@ -631,6 +631,8 @@ reporting.parameter.startDate=Start Date
 reporting.parameter.endDate=End Date
 reporting.parameter.location=Location
 reporting.parameter.locationList=Locations
+reporting.parameter.attributeType=Attribute Type
+reporting.parameter.values=Value
```

Comment:
I would actually think that since the property accepts a List of String values, then we should support the user submitting 1-N values, and call it values to match the property.